### DISCUSSION TITLE
Diary-中国海关博物馆 | Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by giscus** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#338] [Announcements]
# Diary-中国海关博物馆 | Zi-Zi's Journey

来了解一下海关的历史。

https://gz-metal-cell.github.io/2024/07/08/Diary-%E4%B8%AD%E5%9B%BD%E6%B5%B7%E5%85%B3%E5%8D%9A%E7%89%A9%E9%A6%86/

**Comment by promefire** [comment]
海关博物馆为什么不简称”海博“而是”关博“呢🧐

**Reply by GZ-Metal-Cell** [reply]
因为我已经跟 pro 神逛过了天津的海博😭
